pkgs/top-level/python-packages.nix

```diff
@@ -834,6 +834,10 @@ in {
 
   outcome = callPackage ../development/python-modules/outcome {};
 
+  ovirt-engine-sdk = callPackage ../development/python-modules/ovirt-engine-sdk {
+    inherit (pkgs) libxml2Python;
```

Comment:
this is probably not what we want, looks like libxml2Python is built with python2

```
  libxml2Python = let
    libxml2 = python2Packages.libxml2;
  in pkgs.buildEnv { # slightly hacky
    name = "libxml2+py-${res.libxml2.version}";
    paths = with libxml2; [ dev bin py ];
    inherit (libxml2) passthru;
    # the hook to find catalogs is hidden by buildEnv
    postBuild = ''
      mkdir "$out/nix-support"
      cp '${libxml2.dev}/nix-support/propagated-build-inputs' "$out/nix-support/"
    '';
  };
```

Reply:
however, it looks like it takes the propagated build inputs from the `libxml2.dev` derivation, so it might be okay. Just not sure how libxml2 is used in this package